src/main/java/org/apache/accumulo/testing/continuous/ContinuousInputFormat.java

```diff
@@ -158,9 +165,8 @@ public boolean nextKeyValue() {
 
         if (nodeCount < numNodes) {
           CRC32 cksum = checksum ? new CRC32() : null;
-          byte[] prevRow = prevKey != null ? prevKey.getRowData().toArray() : null;
-
           prevKey = currKey;
+          byte[] prevRow = prevKey != null ? prevKey.getRowData().toArray() : null;
```

Comment:
Because of this I was ending up with double the unreferenced counts than I expected when I ran the verification map reduce job.  I will update the docs about expected counts to be more specific.